code/src/components/GameStart.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { game } from '../reducers/game'
+import { generateStartMove } from '../reducers/fetch'
+import { Button, StyledButton } from '../lib/Button'
+import styled from 'styled-components'
```

Comment:
The "import styled from 'styled-components"-line should be on line 3 instead with the other external dependencies. A blank space should be under to separate them from the local dependencies. 🤓